.github/workflows/sweeper.yaml

```diff
@@ -25,6 +25,7 @@ jobs:
           go-version-file: test/hack/resource/go.mod
           check-latest: true
           cache-dependency-path: "test/hack/resource/go.sum"
-      - run: go run main.go
+      # us-east-1 contains the soak tests and resources should be sweept if they are older then 8 days
+      - run: go run main.go ${{ matrix.region == 'eu-north-1' && '-expiration 192h' || '' }}
```

Comment:
Probably just easier to specify this using bash. Something like

```bash
if [[ "${{ matrix.region }}" == "eu-north-1" ]]; then
  go run main.go --expiration 192h
else
  go run main.go --expiration 12h
fi
```

Also, I think it's reasonable to have `12h` be the default expiration time in the script, but it might make sense to just specify it here in the bash cleanup script for clarity so that it's easier to reason about.